code/modules/client/preferences.dm

```diff
@@ -1202,6 +1202,9 @@ GLOBAL_LIST_EMPTY(preferences_datums)
 				if(user.client.prefs.exp[job_title] >= CONFIG_GET(number/senior_timelock)) //If they have more than 50 hours (300 Minutes) past the required time needed for the job, give them access to the senior title
 					for(var/g in J.senior_title)
 						titles_list += g
+				if(user.client.prefs.exp[job_title] >= 600)//100 hours, i can't be fucked making this a config
+					for(var/g in J.ultra_senior_title)
+						titles_list += g
```

Comment:
ultra_senior title is not a list, instead do this:
```suggestion
					if(J.ultra_senior_title)
						titles_list += J.ultra_senior_title
```
Same for senior_title.

Reply:
senior_title is a list here, some roles have multiple